Code/Engine/Foundation/Platform/Android/CrashHandler_Android.cpp

```diff
@@ -0,0 +1,9 @@
+#include <Foundation/FoundationPCH.h>
+
+#if EZ_ENABLED(EZ_PLATFORM_ANDROID)
+#  include <Foundation/Platform/Posix/CrashHandler_Posix.h>
+#endif
+
+
+EZ_STATICLINK_FILE(Foundation, Foundation_Platform_Android_CrashHandler_Android);
```

Comment:
I don't like that for every platform we our now pulling in all other platform cpp files through this mechanism. This just bloats the final binary size for no reason.

Reply:
Not the case anymore.